modules/gapi/src/backends/streaming/gstreamingbackend.cpp

```diff
@@ -339,6 +349,22 @@ void GOCVY::Actor::extractRMat(const cv::MediaFrame& frame, cv::RMat& rmat)
             });
             break;
         }
+        case cv::MediaFormat::GRAY:
+        {
+            std::call_once(m_warnFlag,
```

Comment:
global `once_flag`  means that only single one warning from BGR->NV12, NV12->BGR, GRAY->N12 would be shown

Whence we change source type or use two or more sources it means that only one warning related to single one source would notify us.

What's about three global `once_flags` for each different notifiation?

Reply:
Done

Reply:
Done